usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -225,10 +226,14 @@ def award_id_lookup_pre_delete(self):
         #       returned by a SELECT subquery inside the 'IN').  Thus, this code should return a dataframe directly,
         #       create a temporary view from the dataframe in award_id_lookup_post_delete, and use that temporary
         #       view to either do a subquery in the 'IN' clause or to JOIN against.
+        if not table_exists:
```

Comment:
Instead of having these two checks in these functions that essentially run before everything else, would it make more sense to refactor/extract this check outside them? It could be in the `handle` function after the `if self.etl_level == "initial_run":` and before the `if self.etl_level == "award_id_lookup":`. It would still run before everything and it would be dynamic `etl_level` based like it is in `delete_records_sql` to cover all the scenarios. Also could include a comment explaining why we're doing this check so early on to save time.

Reply:
Where are these "two" checks that you speak of?

Reply:
Are you referring to the table exists check in two different functions?

Reply:
Correct, those two can be refactored to a single check just before the two different functions.

Reply:
Done.